crates/uv-resolver/src/resolver/mod.rs

```diff
@@ -683,15 +691,15 @@ impl<InstalledPackages: InstalledPackagesProvider> ResolverState<InstalledPackag
         forks
```

Comment:
Can't comment that high, but now diverging packages can be empty and the debug message has a hole:

```
DEBUG Splitting resolution on dummy==0.1.0 over  into 3 resolution with separate markers
```

Reply:
Ah I missed this. I'll have this fixed in a follow-up PR.